doc/source/new-project-guide/level-3-tutorial.rst

```diff
@@ -108,23 +208,20 @@ Then, you can run the code by running:
 .. code-block:: bash
 
     python proj_one/reuse_code.py
-    python proj_two/reuse_code.py
 
 
 Run tests
 ^^^^^^^^^^
 
-Notice that the ``tests`` folder contains a test file called ``test_calculator.py``. You can run the tests by running the following command:
+To run tests using ``pytest``, ``cd`` to the top level directory and run the following command:
 
 .. code-block:: bash
 
-    pytest tests/test_calculator.py
+     pytest
 
-Or you can simply run:
+This will run all tests located in the ``tests`` directory.
```

Comment:
yes, this is good.  I also want to teach how you can run pytest on pregressively fewer things (a directory, then a file) but that can also be saved for a higher level.  I am not sure what is the best way to roll that out in the documentation tbh.  Thoughts?

Reply:
OK, let's do it this way.